addons/mail/static/tests/message/link_preview_test.js

```diff
@@ -342,3 +342,31 @@ QUnit.test(
         await contains(".o-mail-Message-bubble");
     }
 );
+
+QUnit.test(
+    "Link preview from the bus should be attached to the corresponding message",
+    async () => {
+        const pyEnv = await startServer();
+        const channelId = pyEnv["discuss.channel"].create({ name: "wololo" });
+        const messageId = pyEnv["mail.message"].create({
+            body: "<a href='linkPreviewLink'>http://linkPreview</a> not empty",
+            message_type: "comment",
+            model: "discuss.channel",
+            res_id: channelId,
+        });
+        const { openDiscuss } = await start();
+        openDiscuss(channelId);
+        pyEnv["bus.bus"]._sendone(pyEnv.currentPartner, "mail.record/insert", {
+            LinkPreview: [
+                {
+                    id: 1,
+                    message: { id: messageId },
+                    image_mimetype: "image/jpg",
+                    source_url:
+                        "https://upload.wikimedia.org/wikipedia/commons/thumb/4/41/Siberischer_tiger_de_edit02.jpg/290px-Siberischer_tiger_de_edit02.jpg",
+                },
+            ],
+        });
```

Comment:
Can't we just reproduce the flow instead of simulating the message coming from the bus?